.ci/scripts/update-otel.sh

```diff
@@ -52,6 +52,6 @@ else
 fi
 elastic-agent-changelog-tool new "$changelog_fragment_name"
 sed -i.bak "s/^kind:.*$/kind: enhancement/" ./changelog/fragments/*-"${changelog_fragment_name}".yaml
-sed -i.bak "s/^summary:.*$/summary: Update OTel components to ${next_contrib}/" ./changelog/fragments/*-"${changelog_fragment_name}".yaml
+sed -i.bak "s/^summary:.*$/summary: Update OTel Collector components to ${next_contrib}/" ./changelog/fragments/*-"${changelog_fragment_name}".yaml
```

Comment:
Can we also change the line that gets the `current_beta_core`?

It's `current_beta_core=$(grep 'go\.opentelemetry\.io/collector/receiver/otlpreceiver ' go.mod | cut -d' ' -f 2 || true)` which is no longer valid since `otlpreceiver` is stable.